### DISCUSSION TITLE
Strided Transposed convolution issue

### DISCUSSION THREAD

**Discussion by EngineerKhan** [jax-ml/jax#8400] [Q&A]
I am trying to implement the strided transposed convolution.

This is my code:

```x = jnp.ones((50,50))
x = jnp.ones((50,50))
y = jnp.ones((3,3))
ans = lax.conv_general_dilated(x[None, None], y[None, None], (1,1),[(0,0),(0,0)],(2,2))
```

But I cannot verify the `ans` size with the formula in section4.5 from [convolution arithmetic](https://arxiv.org/pdf/1603.07285v2.pdf). 
Am I doing anything wrong in using the formula? I have been unable to relate the `conv_general_dilated()`'s formula for transposed convolution properly I think.

**Comment by hawkinsp** [comment]
Have you seen the `lax.conv_transpose` wrapper? https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.conv_transpose.html

(You can look at its implementation to see how it does its padding calculation, if you like.)

**Reply by EngineerKhan** [reply]
I have checked it but it's not allowing me to use the padding (even the default `[(0,0),(0,0)]` I am using here). Can you recommend the equivalent conv_transpose formulation for that?

Also, can you just explain why does it give 97x97 in the above case, rather than 101 by `s(w-1)+f`?
